package.json

```diff
@@ -224,5 +224,8 @@
     "hooks": {
       "commit-msg": "validate-commit-msg"
     }
+  },
+  "dependencies": {
+    "karma-coverage-istanbul-reporter": "^2.0.4"
```

Comment:
This should be in `devDependencies` (I forget to pass `--save-dev` at least 70% of the time I run `npm i`)